packages/dante3/src/styled/menu.ts

```diff
@@ -34,7 +34,7 @@ export const AnchorStyle = styled.div`
       height: 0;
       width: 0;
       position: absolute;
-      left: ${(props) => (props.arrowPosition ? props.arrowPosition : "50%")};
+      left: ${(props) => (props['arrowPosition'] ? props['arrowPosition'] : "50%")};
```

Comment:
why is this change needed?

Reply:
That property wasn't showing up on the props type signature, but the type was indexable by arbitrary strings. I can try typing the styled div with a <{arrowPosition: string}> generic and see if that works. It would certainly be less hacky.